### PR TITLE
Add COVID page

### PR DESCRIPTION
**Pull request by SamLR**
Also move comment blocks in _nav to make it a bit clearer which bits are
commented

### CODE REVIEW

**Review by Jonty — APPROVED** [review]
LGTM, mergeploy

File: templates/about/covid.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<p>We’re continuing to follow the COVID-19 situation closely, and much has changed since we made the <a href="https://blog.emfcamp.org/2020/03/25/emf-2020-cancelled/">decision to cancel EMF 2020</a>.</p>
+
+<p>Much more is likely to change (hopefully for the better) in the run-up to the event, but whatever happens we will be taking steps to ensure the safety of EMF attendees and volunteers.</p>
+
+<p>Given the rapidly changing situation we cannot yet specify what measures will be implemented on site - the COVID precautions we take at EMF will be governed by the prevalence closer to the time, as well as any government restrictions in force.</p>
+
+<p>We will clearly announce the measures in place before the event, and any recommendations we have for attendees to ensure both their safety and the safety of others.</p>
+
+<p>In the meantime we encourage you to get vaccinated & boosted as soon as possible, and test yourself regularly.</p>
```
Comment (davidc) [inline templates/about/covid.html:22]: &amp;

File: templates/about/covid.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% block title %}COVID 19{% endblock %}
+{% block secondary_nav %}
+  {% include "about/_nav.html" %}
+{% endblock %}
+{% set main_class = 'about' %}
+{% block body %}
+<h2>COVID 19</h2>
```
Comment (davidc) [inline templates/about/covid.html:8]: COVID-19 for consistency

File: templates/about/_nav.html [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 <ul role="navigation" aria-label="About">
   {{item("About EMF", "base.about")}}
+  {{item("COVID 19", "base.covid")}}
```
Comment (davidc) [inline templates/about/_nav.html:9]: COVID-19 for consistency

File: templates/about/covid.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+<p><em><small>Revision 1, published 2022-01-12</small></em></p>
+
+<p><strong>This page is under constant review and may change at short notice.</strong></p>
+
+<p>We’re continuing to follow the COVID-19 situation closely, and much has changed since we made the <a href="https://blog.emfcamp.org/2020/03/25/emf-2020-cancelled/">decision to cancel EMF 2020</a>.</p>
```
Comment (davidc) [inline templates/about/covid.html:14]: apostrophe

**Pull request merged**